src/server/dfly_bench.cc

```diff
@@ -35,6 +36,7 @@ extern "C" {
 using std::string;
 
 ABSL_FLAG(uint16_t, p, 6379, "Server port");
+ABSL_FLAG(uint16_t, pool_size, 0, "Number of threads");
```

Comment:
you do not need this flag. any helio-powered backend has `proactor_threads` flag that does the same.
you can run `--help` to see all flags.

Reply:
Don't see this `proactor_threads` argument when running `dfly_bench --help` so i have added it

Reply:
sorry, try `--helpfull`. `--help` shows only the flags from the main module by default.

Reply:
Removed flag